src/Plugin.php

```diff
@@ -2244,11 +2244,11 @@ public function checkGlpiVersion($infos)
         }
 
         if (!$compat) {
-            echo htmlescape(Plugin::messageIncompatible(
+            echo Plugin::messageIncompatible(
```

Comment:
This message and the following are caught inside an output buffer that is later escaped (see `$error .= "<span class='error'>" . htmlescape(ob_get_contents()) . "</span>";` in `src/Glpi/Marketplace/View.php`).

The buffer content can also contain messages outputed by plugins, it is safer to escape its whole content.